src/issues/util.ts

```diff
@@ -365,22 +364,9 @@ export function getUpstreamOrigin(upstream: Remote, resultHost: string = 'github
 	const enterpriseUri = getEnterpriseUri();
 	let fetchUrl = upstream.fetchUrl;
 	if (enterpriseUri && fetchUrl) {
-		// upstream's origin by https
-		if (fetchUrl.startsWith('https://') && !fetchUrl.startsWith('https://github.com/')) {
-			const host = new URL(fetchUrl).host;
-			if (host.startsWith(enterpriseUri.authority) || !host.includes('github.com')) {
-				resultHost = enterpriseUri.authority;
-			}
-		}
-		if (fetchUrl.startsWith('ssh://')) {
-			fetchUrl = fetchUrl.substr('ssh://'.length);
-		}

```

Comment:
This seems to remove more than what the PR title implies, are we losing anything important here/is there unwanted risk after testing has happened?

Reply:
This method is basically trying to get the host of the git remote. The logic here around determining whether it's an ssh or http remote is also done in `Protocol`, and it's done better in `Protocol`. This should have used `Protocol` from the start. 

I tested this with a github.com remote, a GitHub Enterprise remote, an EMU remote, and a ghe.com remote.